ods_tools/main.py

```diff
@@ -171,13 +171,22 @@ def add_exposure_data_args(command):
 
 transform_description = """
 Transform data format to/from OED.
+This transformation can be done either by providing a config file or directly by specifying the input and output files.
+If input and output files are provided, either --oed-air or --air-oed must be specified to indicate the transformation direction.
+
+If a config file is provided, the transformation will be done according to the config file.
+Please note that the config file allows for more options (batch size, file format, database connection, etc.)
 """
 transform_command = command_parser.add_parser('transform', description=transform_description,
                                               formatter_class=argparse.RawTextHelpFormatter)
-transform_command.add_argument('--config-file', help='Path to the config file', required=True)
+transform_command.add_argument('--config-file', help='Path to the config file')
+transform_command.add_argument('--oed-air', help='directly transforms OED to AIR format', action='store_true')
```

Comment:
I don't think those should be direct argument of the command because you will end up having to add n^2 arguments each time you have a new built-in transformation
I would add a single argument named -f --formats 
that tell you which transformation to do
either -f or --config-file needs to be present to do the transformation